spec/models/library_hours_spec.rb

```diff
@@ -22,4 +22,22 @@
       end
     end
   end
+
+  describe '#library_slug' do
```

Comment:
Should there be a test for `location_map` for the context which you are fixing?

Reply:
This is basically testing this. Location map returns a hash which is why I am using library_slug, because library_slug is just pulling a field from location_hash.